<launch>
    <node pkg="ros_nmpt_saliency" type="ros_nmpt_saliency_node" name="ros_nmpt_saliency_node" output="screen">
        <param name="image_topic" value="/usb_cam/image_raw/" />
        <param name="visualize" value="true" />
    </node>
</launch>
